components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -214,6 +257,24 @@ export function CreateWorkspacePage() {
                         {isLoading ? "Loading ..." : isStarting ? "Creating Workspace ..." : "New Workspace"}
                     </Button>
                 </div>
+                {existingWorkspaces.length > 0 && (
+                    <div className="w-full flex flex-col justify-end px-6">
+                        <p className="mt-6 text-center text-base">Running workspaces on this revision</p>
```

Comment:
**question**: I couldn't hold from thinking how this list could be better visualized. What do you think of collapsing the list inside a dropdown?

For running workspaces, I guess this won't be usually more than 2 or 3, but there could be edge cases where this could break the UI by overflowing the height of this feature. Additionally, replacing the primary action button with a block button is something that looks needed here.

| BEFORE | AFTER |
|-|-|
|  <img width="680" alt="Welcome" src="https://user-images.githubusercontent.com/120486/230643894-74100c7d-d467-4ca9-be28-2bb2a6749b9b.png"> | <img width="680" alt="Welcome-1" src="https://user-images.githubusercontent.com/120486/230643895-e453e694-70b2-4c5f-9fbe-b2dd5684909b.png"> |